packages/Main/test/unit/entwine.js

```diff
@@ -112,8 +112,12 @@ describe('Entwine Point Tile', function () {
         });
 
         it('tries to update on the root and fails', function () {
+            const cam = context.camera.camera3D;
+            const originalQuaternion = cam.quaternion.clone();
+            cam.quaternion.identity(); // look away from dataset
             layer.update(context, layer, layer.root);
             assert.strictEqual(layer.root.promise, undefined);
+            cam.quaternion.copy(originalQuaternion);
```

Comment:
The test was (incorrectly) relying on the fact that the camera's world matrix was not updated when calling `Camera.projectBox3PointsInCameraSpace`, thus actually testing a situation where the camera's matrix would be the identity, and so where the camera would not be looking at the dataset.

This became a problem when some functions did update the camera's world matrix (as they should be allowed). In our case, `EffectComposer.render` did that.

Therefore I added the missing matrix update in `Camera.projectBox3PointsInCameraSpace` and for the test I recreated a situation where the camera looks away from the dataset.